src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -323,6 +323,41 @@ public async Task<bool> IsCompatibleAsync(string subject, string schema)
             => await restService.TestLatestCompatibilityAsync(subject, schema).ConfigureAwait(continueOnCapturedContext: false);
 
 
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.IsCompatibleAsync(string, int, string)" />
+        /// </summary>
+        public async Task<bool> IsCompatibleAsync(string subject, int versionId, string schema)
+            => await restService.TestCompatibilityAsync(subject, versionId, schema).ConfigureAwait(continueOnCapturedContext: false);
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.GetCompatibilityAsync(string)" />
+        /// </summary>
+        public async Task<Compatibility> GetCompatibilityAsync(string subject)
+            => await restService.GetCompatibilityAsync(subject).ConfigureAwait(continueOnCapturedContext: false);
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.SetCompatibilityAsync(string, Compatibility)" />
+        /// </summary>
+        public async Task<Compatibility> SetCompatibilityAsync(string subject, Compatibility compatibility)
+            => (await restService.SetCompatibilityAsync(subject, compatibility).ConfigureAwait(continueOnCapturedContext: false)).CompatibilityLevel;
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.GetGlobalCompatibilityAsync()" />
+        /// </summary>
+        public async Task<Compatibility> GetGlobalCompatibilityAsync()
+            => await restService.GetGlobalCompatibilityAsync().ConfigureAwait(continueOnCapturedContext: false);
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.SetGlobalCompatibilityAsync(Compatibility)" />
+        /// </summary>
+        public async Task<Compatibility> SetGlobalCompatibilityAsync(Compatibility compatibility)
```

Comment:
Same comment as above - `UpdateGlobalCompatibilityAsync`, and just return `Task`.